### ISSUE TITLE
Conda does not import external packages with command-line interface

### ISSUE THREAD

**Issue by ozguturgut** [conda/conda#2220]
I am trying to use optimization package (i.e.Gurobi) through Anaconda. (I am not able to import 'nose" either.) This is what I received when I try to run two Python scripts. First one is importing 'gurobipy' and returning an ImportError. Second script is simple code that imports 'numpy' and running without issue. 

![image](https://cloud.githubusercontent.com/assets/17306113/13646047/b863e1fc-e5fb-11e5-9656-78a32c45fbec.png)

Here is results of the info query, what I got from my root folder:

![image](https://cloud.githubusercontent.com/assets/17306113/13646051/bfea7102-e5fb-11e5-8672-a5eafde46c0a.png)

(I have tried Miniconda previously; It had the same issue as well.)

**Comment by kalefranz** [comment]
This is a problem with a package you're using from the public anaconda.org.  Consider contacting that package maintainer.

**Comment by ozguturgut** [comment]
Excuse me.
Can I assume it is certain that this is related with  'package maintainer'; since you closed the issue without letting any other person evaluate it? B/c Gurobi classified this as a  Conda problem.

**Comment by mcg1969** [comment]
Gurobi isn't correct, I am afraid. This is a problem with the package, not conda. I actually worked with them to help develop the original conda package, so feel free to have them reach out to me and perhaps I can assist them.

It does look like you encountered a bug on `conda info` as well, but I do not get the impression that is the key issue here.

**Comment by ozguturgut** [comment]
Thanks @mcg1969 . Gurobi helped me to figure out the issue. Actually it was not related with them either. It was due to multiple installations of Python on my computer. Although I was thinking I would be safe by removing the PYTHONPATH/HOME environment variables, that was not the case obviously. I learned from Dr.Kostja Siefen that I need to run my script with a path addition to my command to make sure I am using the 'Anaconda' version of the python package. When I run my code such as "python script.py" or "...\Anaconda2\python.exe script.py”", I was able to import the package.
